pkg/cloudprovider/nodecontroller/nodecontroller.go

```diff
@@ -136,6 +137,61 @@ func NewNodeController(
 	}
 }
 
+// Generates num pod CIDRs that could be assigned to nodes.
+func generateCIDRs(num int) util.StringSet {
+	res := util.NewStringSet()
+	for i := 0; i < num; i++ {
+		// TODO: Make the CIDRs configurable.
+		res.Insert(fmt.Sprintf("10.244.%v.0/24", i))
```

Comment:
I've added a flag for KUBE_GCE_CLUSTER_CLASS_B and made it configurable. Regarding range /14 for GKE: Tomek is working on GKE support, we will do it in a separate PR. (This PR is for GCE only).